docs/edge.rst

```diff
@@ -0,0 +1,253 @@
+.. _edge:
+
+============
+CrateDB Edge
+============
+
+Crate.io is pleased to `announce CrateDB Edge`_, the hybrid cloud database
+solution integrating local control over regions and data storage with the
+managed software stack from CrateDB Cloud.
+
+The process of getting CrateDB Edge running is less complicated than it may
+seem, thanks to the support for Edge deployment built into the CrateDB Cloud
+Console, our own web UI. Even so, there are some steps involved, and some
+requirements have to be met in order for it to work. This tutorial therefore
+serves as an end-to-end walkthrough of the process and of these prerequisites.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _edge-prereqs:
+
+Prerequisites
+=============
+
+Certain hardware and software specifications must be met in order to make use
+of CrateDB Edge. The most important of these is that you must provide a working
+Kubernetes cluster, one that meets the following requirements:
+
+.. rst-class:: open
+
+* It must contain at least three nodes (for high availability);
+
+* Sufficient CPU per node to run the CrateDB Cloud software stack and the OS;
```

Comment:
Do we have an idea what counts as sufficient CPU?

Reply:
We could provide some information based on the plans we have for cloud. I guess it will widely depend on the expectations. I would say we should mimic the minimum CPU requirements from CrateDB and consider the overhead of the region irrelevant.